recipes/Media/ASR/seq2seq/README.md

```diff
@@ -0,0 +1,50 @@
+# Media ASR with CTC + Seq2Seq models +/- Wav2Vec 2.0.
+
+### Contributors :
+- Gaëlle Laperrière, Avignon University, LIA
+- Yannick Estève, Avignon University, LIA
+- Bassam Jabaian, Avignon University, LIA
+- Sahar Ghannay, Paris-Saclay University, CNRS, LISN
+- Salima Mdhaffar, Avignon University, LIA
+- Antoine Caubrière, Avignon University, LIA
+
+This folder contains scripts necessary to run an ASR experiment with the Media French dataset: [Media ASR (ELRA-S0272)](https://catalogue.elra.info/en-us/repository/browse/ELRA-S0272/), [Media SLU (ELRA-E0024)](https://catalogue.elra.info/en-us/repository/browse/ELRA-E0024/) both needed for the task.
+
+# How to run
+Do not forget to process the dataset and change the `!PLACEHOLDER` in the yaml file.
```

Comment:
How to process the dataset?

Reply:
README in the previous folder for the parseXMLtoSB.py script. Do you want me to say "go read the README" ?